docs/framework/whats-new/obsolete-members.md

```diff
@@ -318,7 +318,7 @@ The tables in this article list the type members that are obsolete in the
 |----------|------------|-------------|  
 |<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.DoDragMove%28System.Activities.Presentation.WorkflowViewElement%2CSystem.Windows.Point%29>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Use                                          <!--zz <xref:System.Activities.Presentation.DragDropHelper.DoDragMove%28System.Collections.Generic.IEnumerable%7BSystem.Activities.Presentation.WorkflowViewElement%7D%2CSystem.Windows.Point%29> -->   `System.Activities.Presentation.DragDropHelper.DoDragMove` instead.|  
 |<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetCompositeView%28System.Windows.DragEventArgs%29>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Use                                          <xref:System.Activities.Presentation.DragDropHelper.GetCompositeView%28System.Activities.Presentation.WorkflowViewElement%29> instead.|  
-|<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetDragDropCompletedEffects%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Consider using                                          <xref:System.Activities.Presentation.DragDropHelper.SetDragDropMovedViewElements%28System.Windows.DragEventArgs%2CSystem.Collections.Generic.IEnumerable%7BSystem.Activities.Presentation.WorkflowViewElement%7D%29> instead.|  
+|<xref:System.Activities.Presentation.DragDropHelper?displayProperty=fullName>|<xref:System.Activities.Presentation.DragDropHelper.GetDragDropCompletedEffects%2A>|First deprecated in the .NET Framework 4.5.<br /><br /> This method does not support dragging multiple items. Consider using <!--zz <xref:System.Activities.Presentation.DragDropHelper.SetDragDropMovedViewElements%28System.Windows.DragEventArgs%2CSystem.Collections.Generic.IEnumerable%7BSystem.Activities.Presentation.WorkflowViewElement%7D%29> -->                                         `System.Activities.Presentation.DragDropHelper.SetDragDropMovedViewElements(System.Windows.DragEventArgs,System.Collections.Generic.IEnumerable{System.Activities.Presentation.WorkflowViewElement})` instead.|  
```

Comment:
@GuardRex, I had meant to address this issue for quite some time, but it's fallen on the back burner. I've finally started to investigate. AFAIK, the recommended alternative method doesn't actually exist in the release versions of the .NET Framework.